forge-gui/res/cardsfolder/upcoming/effluence_devourer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Effluence Devourer
+ManaCost:B R G
+Types:Creature Crocodile
+PT:4/3
+K:Blitz:B R G
+T:Mode$ Sacrificed | ValidCard$ Creature | ValidPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice CARDNAME or another creature, it perpetually gains “{2}, Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery.”
+SVar:TrigEffect:DB$ Effect | ValidCard$ Creature | EffectZone$ Command | StaticAbilities$ PerpetualAbility | Name$ Effluence Devourer's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualAbility:Mode$ Continuous |  Affected$ Creature | AddAbility$ ABToken | Defined$ Remembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ {2} Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery.
```

Comment:
too many spaces before Affected$ (just style issue)

More importantly, this is giving every creature the ability

Defined$ Remembered is not doing anything because that param doesn't do anything here and because you haven't remembered anything